package.json

```diff
@@ -76,10 +78,18 @@
     "posttest": "[ -z \"$TRAVIS\" ] || codecov",
     "codecov": "codecov",
     "bench": "BENCHMARK=true npm run test",
-    "version": "node -e \"console.log(require('./package.json').version)\""
+    "version": "node -e \"console.log(require('./package.json').version)\"",
+    "lint-staged": "lint-staged"
   },
   "dependencies": {
     "is-browser": "^2.0.1",
     "murmurhash-js": "^1.0.0"
-  }
+  },
+  "lint-staged": {
```

Comment:
Sure, another question, it seems like a good name in scripts would be:

```
lint -> lint:all
linst-staged -> lint:staged
```
Makes sense?